modules/ztp-nbde-installing-nbde-with-ztp.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-nbde-implementation-guide.adoc
+
+[id="ztp-nbde-installing-nbde-with-ztp_{context}"]
+= Installing NBDE with ZTP
+
+Zero touch provisioning (ZTP) provides the capability to install Network-Bound Disk Encryption (NBDE) and enable disk encryption at cluster installation through SiteConfig. Use the automated SiteConfig method when you are enabling disk encryption on multiple managed clusters.
+
+You can specify disk encryption with a list of Tang server URLs and associated thumbprints in the site plan that contains the configuration for the site installation. The site plan generates the appropriate corresponding ignition manifest along with the other day-0 manifests and applies them to the hub cluster.
+
+.Example `SiteConfig` custom resource (CR) containing a disk encryption specification
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "site-plan-sno-du-ex"
+  namespace: "clusters-sub"
+spec:
+  baseDomain: "example.com"
+  ...
+  clusters:
+  - clusterName: "du-sno-ex"
+    clusterType: sno
+    clusterProfile: du
+    ...
+    diskEncryption:
+      type: "nbde"
+      tang:
+        - url: "http://10.0.0.1:7500"
+          thumbprint: "1c3wJKh6TQKTghTjWgS4MlIXtGk"
+        - url: "http://10.0.0.2:7500"
+          thumbprint: "WOjQYkyK7DxY_T5pMncMO5w0f6E"
+  …
```

Comment:
```suggestion
  ...
```
We aren't using UTF-8 characters in the docs.

Reply:
This file was commented out of the assembly, so it doesn't appear in the final version. I removed it completely from the assembly.